src/router/router.js

```diff
@@ -0,0 +1,60 @@
+import Vue from 'vue'
+import Router from 'vue-router'
+import CommonLayout from '@/components/layouts/CommonLayout'
+import Home from '@/views/Home'
+import Question from '../views/Question'
+import User from '../views/User'
+import Users from '../views/Users'
+import Tags from '../views/Tags'
+import Tag from '../views/Tag'
+
+Vue.use(Router)
+
+export default new Router({
+  mode: 'history',
+  base: process.env.BASE_URL,
+  routes: [
+    {
+      path: '',
+      component: CommonLayout,
+      children: [
+        {
+          path: '',
+          name: 'home',
+          component: Home
+        },
+        {
+          path: 'question/:id',
+          name: 'question',
+          'params': ['id'],
```

Comment:
why param is as string and not just `params: ['id'],` ?

Reply:
Indeed, I just noticed that. Makes sense because it's just an object key.  Changed to `params`